joomla/joomla-cms
# [4.3] Accessibility Features Plugin: choice of two icon images (Google Font)

Pull Request for Issue #38832 .

### Summary of Changes

- Choice of two icon images (Google Font)
- Choice of a rectangular (default) or a circular shape image

### Testing Instructions

- Enable the 'System - Additional Accessibility Features' plugin
- Select the "Use Google Material Font"
- Test the different options, see the attached images
- Code review

### Actual result BEFORE applying this Pull Request

No option to choose an icon with a shape (Google Font)

### Expected result AFTER applying this Pull Request
![pr_accessibility-2](https://user-images.githubusercontent.com/9271775/212045906-8debae6a-886c-4001-be39-0872a53bbf4e.png)

![pr_accessibility-3](https://user-images.githubusercontent.com/9271775/212045923-0f2b9abf-8eec-4e19-93bc-7d3421fe90aa.png)
![pr_accessibility-4png](https://user-images.githubusercontent.com/9271775/212045996-bf718970-9ae2-4b5a-9fd3-26fb05190c50.png)

### Some notes

See links: 
- https://ranbuch.github.io/accessibility/
- https://ranbuch.github.io/accessibility/site/

It has not researched of the "package": "accessibility", "version": "3.0.17" is still correct.

### Link to documentations
Please select:
